README.md

```diff
@@ -31,7 +31,7 @@ Concretely, there are 3 main packages: `domain`, `use_cases` and `infrastructure
 The `npm` commands are the same, whether you are on `back` or `front` application. 
 
 ### Requirements
-- Node LTS 14.x
+- Node LTS 16.5
```

Comment:
Even though there is a specific version defined in nvmrc for development/testing purposes, I would keep the `.x` in README to have `16.x` because what is important here is the LTS major version.